packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -435,3 +428,22 @@ def _get_enum_class_name(name: str) -> str:
     else:
         name = name[0].upper() + name[1:]
     return f"{name}Enum"
+
+
+def _validate_and_transform_enum_annotations(enum_annotations: str) -> str:
```

Comment:
Please test this functionality. If you can't write an acceptance test, at least write a unit test for this function.

Reply:
Sure! Added unit test for this function.